src/software/visualizer/visualizer_wrapper.h

```diff
@@ -84,6 +88,8 @@ class VisualizerWrapper : public ThreadedObserver<World>,
     static constexpr std::size_t ai_draw_functions_buffer_size    = 2;
     // We only care about the most recent PlayInfo, so the buffer is of size 1
     static constexpr std::size_t play_info_buffer_size = 1;
+    // We don't want to miss any SensorMsg updates so we make the buffer larger
+    static constexpr std::size_t sensor_msg_buffer_size = 60;
```

Comment:
Curious why its 60? Is that anything to do with the rate at which we receive the msg?

Reply:
not sure I just copied the size from the other buffer

Reply:
@MathewMacDougall for his comments on this one, as I imagine he set the buffer size originally.

Reply:
Uh I don't remember why it's 60 (shame on me for no descriptive comment). Likely was just a somewhat arbitrarily medium value. #1300 I think will help mitigate any concerns